scripts/jenkins/ardana/ansible/run-crowbar-tests.yml

```diff
@@ -25,6 +25,16 @@
           debug:
             msg: "http://{{ ansible_host }}:9091/"
 
+        - name: Ensure temporal repo for python-virtualenv is in place
```

Comment:
Nit: I would remove the `temporal` word from here and from the repository name. The fact that we're adding then removing it is enough.